dot-net-backend-home-tasks/Yehor.T/Task2/Form1.cs

```
@@ -0,0 +1,57 @@
+using System.Runtime.CompilerServices;
+using System.Linq.Expressions;
+
+namespace Task2
+{
+  public partial class Form1 : Form
+  {
+    private bool _flag;
+
+    public Form1()
+    {
+      InitializeComponent();
+    }
+
+    private void Form1_Load(object sender, EventArgs e)
+    {
+      maskedTextBox1.ValidatingType = typeof(System.DateTime);
+      maskedTextBox1.TypeValidationCompleted += new TypeValidationEventHandler(MaskedTextBox1_TypeValidationCompleted);
+      maskedTextBox1.TextChanged += MaskedTextBox1_TextChanged;
+    }
+
+    private void MaskedTextBox1_TextChanged(object? sender, EventArgs e)
+    {
+      try
+      {
+        var result = CalcDiff();
+        daysDisplayLabel.Text = result.Days.ToString();
+        hoursDisplayLabel.Text = result.Hours.ToString();
+        minutesDisplayLabel.Text = result.Minutes.ToString();
+        secondsDisplayLabel.Text = result.Seconds.ToString();
+      }
+      catch (FormatException _) { }
```

Not a best choice to hide Exceptions, typicaly they written to some Log or Registry for further analysis.